docs/source/index.rst

```diff
@@ -98,7 +104,7 @@ Developer Setup
 Overview
 --------
 
-To be written.
+To be written. 
```

Comment:
I know this is tedious, but either complete this from the paper? Or remove this section?